transport/http/config_test.go

```diff
@@ -56,9 +56,10 @@ func TestTransportSpec(t *testing.T) {
 	}
 
 	type wantInbound struct {
-		Address    string
-		Mux        *http.ServeMux
-		MuxPattern string
+		Address     string
+		Mux         *http.ServeMux
+		MuxPattern  string
+		GrabHeaders map[string]bool
```

Comment:
nit: when using maps as sets, `struct{}` is recommended over `bool`.

    map[string]struct{}

It avoids accidentally misusing the value as the check.

Reply:
Fixed